src/middlewared/middlewared/plugins/sysctl/hint_isp_role_freebsd.py

```diff
@@ -0,0 +1,28 @@
+import sysctl
+
+
+async def setup(middleware):
+    if not await middleware.call("keyvalue.get", "run_migration", False):
+        return
+
+    if await middleware.call("system.is_ix_hardware"):
+        return
+
+    if await middleware.call("keyvalue.get", "hint_isp_role_tunables_created", False):
+        return
+
+    tunables = {tunable["var"] for tunable in await middleware.call("tunable.query", [["var", "^", "hint.isp."]])}
+    for i in range(0, 4):
+        var = f"hint.isp.{i}.role"
+        if var in tunables:
+            continue
+        if not sysctl.filter(var):
+            continue
+
+        await middleware.call("tunable.create", {
+            "var": var,
+            "value": "2",
+            "type": "SYSCTL",
```

Comment:
The "hint.*" are only tunables, not sysctl's.  Respective sysctl's are called "dev.isp...", but in this case use of tunables is preferable to make disks probed at boot.

Reply:
@amotin so they'll be `LOADER` tunables and we'll need a reboot to properly apply them?

Reply:
Ah right. Thats why I said we should prob do that at install time and on upgrade phase, so the extra reboot is not necessary